docs/assignment1.md

```diff
@@ -1,62 +1,154 @@
+attack the player back. If the attack hits there is 50/50 chance of damage. The combat system should have a modifier mechanic on which the chance
+of hitting or damage depends on. "Stronger" enemies have higher modifiers thus will give more damage to the player. </td>
+</tr>
+<tr>
+<td>F8</td>
+<td>Inventory</td>
+<td>The game shall have a simple <b>inventory system</b> using slots for weapons, currency and other items. Simple means there is no weight and volume system.
+Weapons are not stackable, while currency and other items are.</td>
+</tr>
+<tr>
+<td>F9</td>
+<td>Time Mechanic</td>
+<td>The game shall have a <b>time system</b> for story-telling purposes. Time will be implemented with a turn-based system, for example: every <i>x</i> turns
+<i>y</i> amount of time passes. Some actions will pass time immediately, such as sleeping.</td>
+</tr>
+</table>
 
 ### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
-
-As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
+Author(s): Claudia Grigoras
 
 | ID  | Short name  | Quality attribute | Description  |
 |---|---|---|---|
-| QR1  | Commands sanity checks | Reliability  | When the player issues a command, the syntax of the command shall always get validated against the format specified in F2 |
-| QR2  | Extensible world | Maintainability  | The video game shall be easilty extendable in terms of levels, worlds, interaction points  |
-| QR3  | Instantaneous results | Responsiveness  | Once the scores of all soccer players are provided by the user, the results of the virtual matches shall be available within 1 second |
-| QR4  | ... | ... | ... |
+| QR1  | Commands sanity checks | Reliability | When the player issues a command, the syntax of the command shall always get validated against the format specified in F3 |
+| QR2  | Adaptable system | Maintainability & Responsiveness | The video game shall be easily adaptable to new stories and worlds. The system should not be dependent to one story only (when changing the story or the world, only variables and actions should change) |
+| QR3  | Loading game state | Responsiveness | Once the player types in a command, loading the game state shall be executed in less than 5 seconds regardless of the computer capabilities |
+| QR4  | Accessing the commands | Usability & Availability | The players shall at any point within the game be able to access the available commands at that particular level in game (through the command specified at F3 – examine\) |
+| QR5  | Secured connections | Security | When playing in a multiplayer mode, the system should provide a safe network connection to both user  |
+| QR6  | Real-time commands multiplayer mode | Responsiveness & Reliability | The system shall have a maximum latency of 2 seconds for providing commands and loading game state between 2 players in multiplayer mode |
```

Comment:
Latency could fluctuate in a multiplayer scenario because it is tied to the specific user too, what if said user's internet connection slows down to a halt? When you say shall, it shows that you will make it be 2 seconds or less in any scenario. It stands better to assume that it *should* be 2 seconds or less assuming regular circumstances. The second part is slightly ambiguous, do you mean loading a scene and it's objects?